website/docs/ref/enums.doc.js

```diff
@@ -0,0 +1,78 @@
+/* @flow */
+/*
+---
+id: enums
+title: Enums
+permalink: /docs/enums.html
+prev: react.html
+---
+*/
+
+/*
+In Flow you can [represent enums using literal types](builtins.html#literal-types).
```

Comment:
Mention union types instead so it says "In Flow you can use union types similar to enums" (okay to use the word enums here, but distance from it, better not to have people calling these enums).